improver/utilities/cube_manipulation.py

```diff
@@ -772,3 +772,53 @@ def maximum_in_height(
         max_cube.rename(new_name)
 
     return max_cube
+
+
+def height_of_maximum(
+    cube: Cube,
+    max_cube: Cube,
+    high_or_low: str = 'low',
+    new_name: str = None,
+) -> Cube:
+    """Calculates the height level at which the maximum value has been calculated. This
+    takes in a cube with values at different heights, and also a cube with the maximum
+    of these heights. It compares these (default is to start at the lowest height and 
+    work down through the height levels), and then outputs the height it reaches the maximum value.
+
+    Args:
+        cube:
+            A cube with a height coordinate.
+        max_cube:
+            A cube of the maximum value over the height coordinate.
+        high_or_low: 
+            Whether we are looking for the highest or lowest maximum height (for the case where the
+            maximum occurs at more than one height)
+        new_name:
+            The new name to be assigned to the output cube. If unspecified the name of the original
+            cube is used.
+    Returns:
+        A cube of heights at which the maximum values occur.
+
+    Raises:
+        ValueError:
+            If the cube has only 1 height level or if an input other than high or low is tried for the 
```

Comment:
I've changed this - it will raise an error now. I think I accidentally deleted the test for it last time. I haven't changed it to a boolean, it still does a ValueError. I'm not sure it really matters?

Reply:
It only means that you don't have to raise a custom error as you restict the variable to being true or false and Python will handle the error if anything else is entered. I don't think it matters too much here.